src/components/PivotTable/PivotTableContainer.js

```diff
@@ -1,19 +1,41 @@
+import classnames from 'classnames'
 import PropTypes from 'prop-types'
 import React from 'react'
+import { usePivotTableEngine } from './PivotTableEngineContext'
 import { table as tableStyle } from './styles/PivotTable.style'
 
 export const PivotTableContainer = React.forwardRef(
-    ({ width, height, children }, ref) => (
-        <div
-            className="pivot-table-container"
-            style={{ width, height }}
-            ref={ref}
-            data-test="visualization-container"
-        >
-            <style jsx>{tableStyle}</style>
-            {width === 0 || height === 0 ? null : <table>{children}</table>}
-        </div>
-    )
+    ({ width, height, children }, ref) => {
+        const engine = usePivotTableEngine()
+
+        return (
+            <div
+                className="pivot-table-container"
+                style={{ width, height }}
+                ref={ref}
+                data-test="visualization-container"
+            >
+                <style jsx>{tableStyle}</style>
+                {width === 0 || height === 0 ? null : (
+                    <table
+                        className={classnames({
+                            'fixed-headers':
+                                engine.options.fixColumnHeaders &&
+                                engine.options.fixRowHeaders,
+                            'fix-column-headers':
```

Comment:
why both `fixed` and `fix`? use `fixed` everywhere?

```suggestion
                            'fixed-column-headers':
```